e2e/playwright/viewer/src/tests/viewer.spec.ts

```diff
@@ -76,10 +76,10 @@ test.describe('viewer file', () => {
     await personalFiles.dataTable.performClickFolderOrFileToOpen(randomDocxName);
     expect(await personalFiles.viewer.isViewerOpened(), 'Viewer is not opened').toBe(true);
     await personalFiles.viewer.closeButtonLocator.click();
-    expect(await personalFiles.viewer.isViewerOpened(), 'Viewer did not close').toBe(false);
+    expect(await personalFiles.viewer.viewerLocator.isVisible(), 'Viewer did not close').toBe(false);
```

Comment:
you already have `isViewerOpened` that checks for visibility, no need making the `viewerLocator` public

Reply:
Mark viewerLocator as private.